src/ServicePulse.Host/app/js/services/service.formatter.js

```diff
@@ -4,22 +4,32 @@
     function formatter() {
         var secondDuration = moment.duration(1000);
         var minuteDuration = moment.duration(60 * 1000);
-        var hourDuration = moment.duration(60 * 60 * 1000);
+        var hourDuration = moment.duration(60 * 1000); //this ensure that we never use minute formatting
         var dayDuration = moment.duration(24 * 60 * 60 * 1000);
 
         function formatTime(value) {
             var duration = moment.duration(value);
 
+            var time = { value: 0, unit: '' };
             if (duration >= dayDuration) {
-                return duration.format('D [d] h [h]');
+                time.value = duration.format('D [d] h [h]');
+                return time;
             } else if (duration >= hourDuration) {
-                return duration.format('hh:mm [h]');
+                time.value = moment(duration._data).format('HH:mm');
```

Comment:
Why is this the only one accessing the `_data` property?

Reply:
The issue was that without it, when you use format HH:mm the hours will be skipped if equal to 00, leaving the value of for example: `34`.

This hack was suggested in few places for example here: https://stackoverflow.com/a/41443446